confd/warp/warp.go

```diff
@@ -0,0 +1,228 @@
+package warp
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	. "github.com/cloudogu/ces-confd/confd"
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
should not use dot imports